package.json

```diff
@@ -25,7 +25,13 @@
     }
   },
   "scripts": {
-    "preinstall": "yarn --cwd ccd-definition-processor install",
+    "preinstall": "git submodule update --init --recursive && yarn --cwd ccd-definition-processor install",
+    "lint": "echo lint",
+    "test": "mocha test/**/*.js",
+    "test:coverage": "echo test:coverage",
+    "test:a11y": "echo test:a11y",
+    "sonar-scan": "node_modules/sonar-scanner/bin/sonar-scanner",
+
```

Comment:
Most of the above are just sample scripts. We will eventually introduce `lint`, `test:coverage`, etc where applicable. These are required for now as the pipeline expects those as a minimum for a nodejs project